src/main/java/webserver/Routes.java

```diff
@@ -33,8 +39,14 @@ public Routes(Controller controller) {
     }
 
     public String get(String path) throws IOException {
-        final String response = getRoutes.get(path);
-        return  response != null ? response : controller.getNotFound();
+        if (getRoutes.containsKey(path)) {
+            final String response = getRoutes.get(path);
+            return response != null ? response : controller.getNotFound();
```

Comment:
Is response ever null here?  It looks like you're already checking `#containsKey` a couple lines above, so the `Map` should have `path` in it as long as it's not being concurrently modified in another thread.